src/commonmark/en/content/developer/web-api/new-tracker.md

```diff
@@ -0,0 +1,2046 @@
+# New Tracker
+
+Version 2.36 of DHIS2 introduced a set of new tracker endpoints dedicated to importing and querying tracker objects (Including tracked entities, enrollments, events, and relationships).
+These new endpoints set a discontinuity with earlier implementations. Re-engineering the endpoints allowed developers to improve, redesign, and formalize the API's behavior to improve the Tracker services.
+
+The newly introduced endpoints consist of:
+* `POST /api/tracker`
+* `GET /api/tracker/enrollments`
+* `GET /api/tracker/events`
+* `GET /api/tracker/trackedEntities`
+* `GET /api/tracker/relationships`
+
+Significant changes occurred in version 2.36 to make the interface with clients homogeneous and to allow more consistent and flexible ways to use the services.
```

Comment:
This feels like we are unnecessarily filling space :). `Re-engineering the endpoints allowed developers to improve, redesign, and formalize the API's behavior to improve the Tracker services.` should be enough.